data/config.example.toml

```diff
@@ -1,11 +1,12 @@
 [segfautils]
 port = 6893
 auth_token = "YOURAUTHTOKEN"
+shoutrrr_url = "YOURSHOUTRRURL"
```

Comment:
No

Reply:
Why not?

Reply:
I don't see where the link is pointing to

Reply:
I would rather have a [Service URL](https://containrrr.dev/shoutrrr/v0.5/services/overview/) as the default example.

Reply:
hmm, and how do you think you can implement that?

Reply:
``shoutrrr_url = "discord://token@id" # Shoutrrr URL examples are available at https://containrrr.dev/shoutrrr/v0.5/services/overview``